### ISSUE TITLE
Could not find a declaration file for module '@devexpress/dx-grid-core'.

### ISSUE THREAD

**Issue by Methuselah96** [DevExpress/devextreme-reactive#2373]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Current Behaviour

```
TS7016: Could not find a declaration file for module '@devexpress/dx-grid-core'. 'C:/Users/ParagonUser/Documents/ParagonCore/node_modules/@devexpress/dx-grid-core/dist/dx-grid-core.umd.js' implicitly has an 'any' type.
  Try `npm install @types/devexpress__dx-grid-core` if it exists or add a new declaration (.d.ts) file containing `declare module '@devexpress/dx-grid-core';`
```

### Expected Behaviour

It compiles successfully.

### Steps to Reproduce

Install `@devexpress/dx-react-grid@2.1.0`.

### Environment

- devextreme-reactive: 2.1.0
- react: 16.10.1
- browser: N/A
- bootstrap: N/A
- react-bootstrap: N/A
- material-ui: 4.4.3

**Comment by ushkal** [comment]
Hi,

Thank you for reporting the problem. We'll fix it soon.

**Comment by ushkal** [comment]
We've fixed this issue. The fix is available in version 2.1.1.
